config/application.rb

```diff
@@ -9,11 +9,14 @@
 module Directory
   class Application < Rails::Application
     # Initialize configuration defaults for originally generated Rails version.
-    config.load_defaults 6.0
+    config.load_defaults 6.1
```

Comment:
This is a bit different than we've been doing this elsewhere, but I think for this app it probably doesn't matter.